x-pack/plugins/security_solution/common/detection_engine/utils.test.ts

```diff
@@ -113,3 +113,26 @@ describe('#hasNestedEntry', () => {
     });
   });
 });
+
+describe('#hasEqlSequenceQuery', () => {
+  describe('when a non-sequence query is passed', () => {
+    const query = 'process where process.name == "regsvr32.exe"';
+    it('should return false', () => {
+      expect(hasEqlSequenceQuery(query)).toEqual(false);
+    });
+  });
+
+  describe('when a sequence query is passed', () => {
+    const query = 'sequence [process where process.name = "test.exe"]';
+    it('should return true', () => {
+      expect(hasEqlSequenceQuery(query)).toEqual(true);
+    });
+  });
+
+  describe('when a sequence query is passed with extra white space', () => {
```

Comment:
thanks for the updates.
how does this work for `"sequence\n..."`? or `"sequence where ..."` which _should_ be invalid syntax on the ES side. but if it is accepted syntax, then it's definitely not a sequence query

Reply:
added coverage for `\n` but it should work for the `sequence where ...` case already. Added another test for coverage sake